scripts/deployment-test/netlify.mjs

```diff
@@ -27,6 +30,12 @@ async function createNewApp() {
     useTypeScript: true,
     projectDir: PROJECT_DIR,
   });
+
+  // In conjunction with NETLIFY_EDGE_CI we explicitly call remix init
+  // to create the edge version of the app
+  if (typeof NETLIFY_EDGE_CI !== "undefined") {
```

Comment:
```suggestion
  if (process.env.USE_NETLIFY_EDGE) {
```

Reply:
This won't work if you have `true` and `false` as suggested above because they are converted to strings, so `"false"` and `"true"` will both evaluate to `true`. Mind you with my value of `1` which converts to `"1"` this would work. Not sure why I didn't do it that way as I normally do existence checks instead of `typeof`.